enterprise-suite/templates/resourcelimits.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: {{ .Release.Namespace }}-namespace-defaults
+spec:
+  limits:
+  - defaultRequest:
+      cpu: {{ .Values.defaultCPURequest }}
+      memory: {{ .Values.defaultMemoryRequest }}
+    type: Container
```

Comment:
Isn't this redundant with specifying the default request on each pod?

Reply:
Probably!  Just leaving as a kind of bookmark in case someone wants a thing there.